src/web/modules/transactions/transactions.http.ts

```diff
@@ -122,12 +126,18 @@ export async function show(req: Request, res: Response, next: NextFunction): Pro
     }
 
     const renderKey = transaction.transaction_type.toLowerCase()
+
+    if (transaction.gateway_transaction_id && transaction.gateway_transaction_id.match(stripe_payment_re)) {
```

Comment:
We could look at the payment provider rather than using a regex? This will work fine, but what if Stripe change the pattern - probably unlikely though